appjs/views/mixins/form.js

```diff
@@ -73,21 +73,31 @@ module.exports = {
 
         return inst.save();
       }).then(function() {
-        logger.debug('form finished saving');
-
         if ( action === 'new' ) {
           app.view.success('New '+model_class+' saved');
         } else {
-          app.view.success(model_class+' updates saved');
+          if(!app.view.findNotification('Publishing...')){
+            if(model_class === 'Project'){
+              if(view.model.hasUnpublishedUpdates()){
+                app.view.info(model_class+' updates saved but not published');
+              } else {
+                app.view.success(model_class+' updates saved');
+              }
+            } else {
+              app.view.success(model_class+' updates saved');
+            }
+          }
```

Comment:
Prevents overlapping notification on project save/build. If the project is `publishing`, the alert status will remain as "Publishing..." and not be overridden by "updates saved."